### PR TITLE
🐛 Fix unmarshal for terraform plan `replace_paths`

### PR DESCRIPTION
**Pull request by czunker**
Fixes https://github.com/mondoohq/cnspec/issues/689

### CODE REVIEW

File: resources/packs/terraform/tfplan.go [hunk 1]
```diff
@@ -84,7 +84,7 @@ func (t *mqlTerraformPlan) GetResourceChanges() ([]interface{}, error) {
 			}
 		}
 
-		var replacePaths map[string]interface{}
+		var replacePaths []interface{}
```
Comment (chris-rock) [inline resources/packs/terraform/tfplan.go:87]: Great. Can we add a test for that specific case?

**Comment by czunker** [comment]
This is, how it looks after the change:
```
cnquery run terraform plan /tmp/tf-plan.json -c "terraform.plan.resourceChanges[0].change.replacePaths" 
→ loaded configuration from /etc/opt/mondoo/mondoo.yml using source default
→ discover related assets for 1 asset(s)
→ resolved assets resolved-assets=1
terraform.plan.resourceChanges[0].change.replacePaths: [
  0: [
    0: "member"
  ]
]
```

File: resources/packs/terraform/tfplan.go (hunk 1 above)
Comment (czunker) [reply resources/packs/terraform/tfplan.go:87]: Sure. I added one to check this resource.

**Review by chris-rock — APPROVED** [review]
Thank you @czunker

**Pull request merged**